src/VirtualClient/VirtualClient.Dependencies/AMDGPUDriverInstallation.cs

```diff
@@ -107,17 +141,192 @@ await this.stateManager.SaveStateAsync(nameof(AMDGPUDriverInstallation), new Sta
                     }
                     else if (this.Platform == PlatformID.Unix)
                     {
-                        throw new DependencyException(
-                                $"AMD GPU Driver Installation is not supported by Virtual Client on the current platform '{this.Platform}'",
-                                ErrorReason.PlatformNotSupported);
+                        LinuxDistributionInfo linuxDistributionInfo = await this.systemManager.GetLinuxDistributionAsync(cancellationToken)

```

Comment:
There are more than 2 calls to GetLinuxDistributionAsync. COuld we do it in initialization and save it as a private variable?